stable/tomcat/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
This is an outdated `_helpers.tpl`. Please compare with what `helm create` using Helm 2.8+ gives you and update the whole chart accordingly.

Reply:
I refactor the entire chart based on the `helm create` template. Thanks for the tip